### ISSUE TITLE
wishlist: 

### ISSUE THREAD

**Issue by WhiteChairFromIkea** [notepad-plus-plus/notepad-plus-plus#10086]
### Description of the Issue

"Backup path" field under "Session snapshot and periodic backup" panel is read only. It should be editable:

![image](https://user-images.githubusercontent.com/46539653/123923604-d6abca00-d991-11eb-84b7-ed64506d2e67.png)

### Steps to Reproduce the Issue

1. Open Settings -> Preferences -> Backup
2. Under "Backup on save" panel: Select "Verbose.." -> Check "Custom backup directory" -> Enter any path into "Directory:" field. This field also accepts path, containing windows environment variable, like %tmp% (resolves to C:\Users\appdata->Temp" or sometjing like that) or %appdata%; 
3. Try to edit "Backup path:" field under "Session snapshot and periodic backup" in the same way. This path field is not editable, path cannot be changed.

### Expected Behavior

It is expected for "Backup path" field under "Session snapshot and periodic backup" panel to be editable in the same fashion as "Directory" field under "Backup on save" panel. It could accept environment variables.

### Debug Information
```
Notepad++ v8.1   (64-bit)
Build time : Jun 17 2021 - 01:56:07
Path : ...\Notepad++\notepad++.exe
Command Line : ...
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Home (64-bit) 
OS Version : 2009
OS Build : 19042.1052
Current ANSI codepage : 1252
Plugins : IndentByFold.dll LocationNavigate.dll MenuIcons.dll NppConverter.dll NppFTP.dll NppQrCode64.dll NppXmlTreeviewPlugin.dll PythonScript.dll RunMe.dll _CustomizeToolbar.dll 
```

**Comment by pryrt** [comment]
Another use-case for this is when a user (like the forum's [@Jürgen-Kortt](https://community.notepad-plus-plus.org/topic/21399/change-session-backup-path-without-changing-cloud-sync)) wants to have the settings on the cloud, but doesn't want the periodic backup on the cloud (because a periodic backup of a 100Mb file that's being edited will use up a lot of your cloud-storage quota.

* reference = https://community.notepad-plus-plus.org/topic/21399/change-session-backup-path-without-changing-cloud-sync

**Comment by MimirRnD** [comment]
The above function is important to implement.
I am using several computers. One for stationary use and one for travel
When I open the notepad on the travel laptop my server is syncing the files to the laptop. If I use the current version it cannot sync. I must copy the notepad files manually

It does not make sense to even have the temp files in a hidden folder for windows/roaming at all even. Make it editable

**Comment by Googolplexed06** [comment]
Bump! I definitely want this feature to be implemented. I use Cloud & Link to save my Notepad++ config non-locally, and would prefer my current session only be saved locally.

**Comment by menasheungar** [comment]
I use OneDrive, but it is the same for everybody using Dropbox or whatever. 
You want every file you open to be available from any other computer you use in a different location but connected to the same cloud service like Onedrive or Dropbox.
In short the first backup location MUST be editable so it points to a shared location.

**Comment by Kjcavin** [comment]
Agreed, customized session backup paths is truly a must. I understand the argument that if you want them saved just save the doc to the cloud drive. However, if you're anything like me you can open a handful of new docs at once while troubleshooting things that you can't resolve right away. So I rely heavily on the session backups and would be screwed if my PC were to die before I could save them (which is unfortunately, likely to happen at some point).

**Comment by minig0d** [comment]
Similar reasons to all the above... Multiple computers at work / home that I want to share same basics so I don't need to manually configure them all, but can't have proprietary or confidential stuff getting synced to my cloud storage.  I assume it was likely linked for convenience during development (or just assuming there wasn't a need to be able to customize it), but really can't think of a technical reason why this couldn't be implemented in a few minutes.

It's got my vote.
